necromorphs/code/modules/weapons/guns/divet.dm

```diff
@@ -1,50 +1,235 @@
+/**
+Divet pistols
+*/
+
 /obj/item/gun/ballistic/automatic/pistol/divet
 	name = "divet pistol"
 	desc = "A Winchester Arms NK-series pistol capable of fully automatic fire."
-	icon = 'necromorphs/icons/obj/weapons/gun.dmi'
+	icon = 'necromorphs/icons/obj/weapons/ds13guns.dmi'
```

Comment:
What is the point of renaming the dmi? It is in the dead space folder already

Reply:
![image](https://github.com/DS-13-Dev-Team/DS13-2.0/assets/104938566/3706b398-4a8d-45f8-a115-62c807a32be1)

Reply:
Was just named "guns" before hand. Made it "ds13guns" to work with the other file names